glmnet/logistic.py

```diff
@@ -226,7 +232,7 @@ def fit(self, X, y, sample_weight=None, relative_penalties=None):
 
         return self
 
-    def _fit(self, X, y, sample_weight=None, relative_penalties=None):
+    def _fit(self, X, y, sample_weight=None, relative_penalties=None, max_features=None):
```

Comment:
I'm not sure this function needs `max_features` here. Thoughts?

Reply:
Good point!